src/app/static/src/app/components/adr/ADRKey.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+    <div class="row mb-3">
+        <div class="col-8">
+            <div class="d-flex">
+                <label for="key"
+                       class="font-weight-bold align-self-stretch"
+                       >ADR API Key: </label>
+                <div class="align-self-stretch pl-2">
+                    <div v-if="!editing">
+                        <span class="pr-2">{{keyText}}</span>
+                        <a href="#" v-if="!key" @click="edit">add</a>
```

Comment:
Using falsiness here is nice because I guess if the user hadn't spotted 'remove' they might just delete the whole key and I guess end up with empty string rather than null, which should behave as if 'none provided'. Worth a unit test?

Reply:
I've actually added a check to stop them from saving an empty string:
https://github.com/mrc-ide/hint/pull/276/commits/e43ff8d8007ff9ef54c4bf3f2ba56c7e1f67106f

Nb, we'll probably expand this at some point to get some validation of the key by making a request with it to ADR.